yoti-sdk-api/src/main/java/com/yoti/api/client/shareurl/policy/DynamicPolicy.java

```diff
@@ -260,11 +276,34 @@ public Builder withWantedRememberMeOptional(boolean wantedRememberMeOptional) {
             this.wantedRememberMeOptional = wantedRememberMeOptional;
             return this;
         }
+
+        public Builder withIdentityProfile(IdentityProfile identityProfile) {
+            this.identityProfile = identityProfile;
+            return this;
+        }
         
         public DynamicPolicy build() {
-            return new DynamicPolicy(wantedAttributes.values(), wantedAuthTypes, wantedRememberMe, wantedRememberMeOptional);
+            return new DynamicPolicy(
+                    wantedAttributes.values(),
+                    wantedAuthTypes,
+                    wantedRememberMe,
+                    wantedRememberMeOptional,
+                    identityProfile
+            );
         }
 
     }
 
+    private static final class Property {
```

Comment:
I generally like this (pushing magic strings and magic numbers down to a single source of truth / constant / static final etc), however, I do not remember seeing the same pattern being used in the java SDK for these @JsonProperty("X") declarations. I believe each field here is only being used once (so we are not making anything DRY-er), and seeing as the only difference between the field names and the string values is the casing - I wonder if for consistency and simplicity we would not be better off in this case simply using the relevant strings for each JsonProperty() directly?

Reply:
I see this pattern in use elsewhere in the Java SDK code already, so marking this as resolved.

Reply:
This is more a team's mindset/style. In this specific scenario is not really required as it is used only once but usually we De/Serialize JSONs using the same POJO.

Reply:
Sure - makes perfect sense.